### ISSUE TITLE
How to make this package runnable with GitHub Action

### ISSUE THREAD

**Issue by LaoshuBaby** [osmlab/name-suggestion-index#5121]
At the comment at https://github.com/osmlab/name-suggestion-index/commit/263678b5583ee4c45c95a18db13b83542ad26eb2 I occured an idea that we can write the manual release workflow into GitHub Action.

So I use this fork to test and try to write this yml files

There have some problems that I meet

## 1. ```run -s ``` can't work, you need to separate it into single command when using GitHub Action

You can see the result at https://github.com/LaoshuBaby/name-suggestion-index/runs/2818469703
![图片](https://user-images.githubusercontent.com/42690037/121875240-d0371480-cd3a-11eb-904e-faac6f6414b1.png)
At this time the ```release.yml``` is 
https://github.com/LaoshuBaby/name-suggestion-index/blob/e7d61ebcee9f8328282ee82f59e1966c054eaca1/.github/workflows/release.yml#L27 
and the ```package.json``` have no different with [origin repo.](https://github.com/osmlab/name-suggestion-index/blob/15c4f055a81ca3ff38f87b4d5264904f3cb3f711/package.json)

## 2. ```npm-run-all -s``` is not supported, even if install ```npm i run-s```

You can see the result at https://github.com/LaoshuBaby/name-suggestion-index/runs/2818493344
![图片](https://user-images.githubusercontent.com/42690037/121875487-17250a00-cd3b-11eb-8bc2-a8848c58d78b.png)
At this time the ```release.yml``` is the same as above
and the ```package.json``` is 
https://github.com/LaoshuBaby/name-suggestion-index/blob/a36274749049300bcec102f899286d52d0a3343d/package.json#L73

## 3. A yml without ```npm i run-s``` will failed in ```run dist:*```

You can see the result at https://github.com/LaoshuBaby/name-suggestion-index/runs/2818714804 for failed one
![图片](https://user-images.githubusercontent.com/42690037/121875890-8995ea00-cd3b-11eb-93ef-c6c4c95fd163.png)
At this time the ```release.yml``` is 
https://github.com/LaoshuBaby/name-suggestion-index/blob/bdb2b078c347ed01649806a672be8b11e8abf755/.github/workflows/release.yml#L27 
and the ```package.json``` have no different with [origin repo.](https://github.com/osmlab/name-suggestion-index/blob/15c4f055a81ca3ff38f87b4d5264904f3cb3f711/package.json)

And see the result at https://github.com/LaoshuBaby/name-suggestion-index/runs/2818641134 for succeed one
![图片](https://user-images.githubusercontent.com/42690037/121875843-7daa2800-cd3b-11eb-9a85-c49082700f3b.png)
At this time the ```release.yml``` is 
https://github.com/LaoshuBaby/name-suggestion-index/blob/8ec8ee660987e07c9235baac9f72bbf82423e8f6/.github/workflows/release.yml#L27 
and the ```package.json``` have no different with [origin repo.](https://github.com/osmlab/name-suggestion-index/blob/15c4f055a81ca3ff38f87b4d5264904f3cb3f711/package.json)

**Comment by bhousel** [comment]
It looks like you're missing the `npm install` step?
Compare with how the build workflow looks:
https://github.com/osmlab/name-suggestion-index/blob/main/.github/workflows/build.yml

**Comment by LaoshuBaby** [comment]
> 
> 
> It looks like you're missing the `npm install` step?
> Compare with how the build workflow looks:
> https://github.com/osmlab/name-suggestion-index/blob/main/.github/workflows/build.yml

Emmm……But the workflow in our production repo only have a ```uses: actions/setup-node@v1``` to setup nodejs environment.
I will try to install before ```run dist:*``` and ```run wikidata```.(But you can see the last picture, if we ignore the unfixed wikidata problem, the run is successful)

(Besides, I still can't push to github but I haven't know where the bug is. I'll debug tomorrow)

**Comment by LaoshuBaby** [comment]
> I will try to install before `run dist:*` and `run wikidata`.(But you can see the last picture, if we ignore the unfixed wikidata problem, the run is successful)
> 
> (Besides, I still can't push to github but I haven't know where the bug is. I'll debug tomorrow)

https://github.com/LaoshuBaby/name-suggestion-index/runs/2821782023

In this(https://github.com/LaoshuBaby/name-suggestion-index/commit/1c9d3f69195ca34a8062a1792e8c73e52f3b039b) version of [release.yml](https://github.com/LaoshuBaby/name-suggestion-index/blob/1c9d3f69195ca34a8062a1792e8c73e52f3b039b/.github/workflows/release.yml), run is OK

If use ```npm install```, the install step may take more than 5 minutes, that will cause github kill this run.

**Comment by bhousel** [comment]
> If use npm install, the install step may take more than 5 minutes, that will cause github kill this run.

I don't think so - our CI builds are already doing this, and the `npm install` step only takes about 30sec

https://github.com/osmlab/name-suggestion-index/blob/54c8069a11ad202e53a3325dfb136af7a338073b/.github/workflows/build.yml#L21-L28

<img width="400" alt="Screen Shot 2021-06-14 at 12 32 04 PM" src="https://user-images.githubusercontent.com/38784/121927029-96e7b000-cd0c-11eb-8ca7-c32f058742cc.png">

**Comment by LaoshuBaby** [comment]
> > If use npm install, the install step may take more than 5 minutes, that will cause github kill this run.
> 
> I don't think so - our CI builds are already doing this, and the `npm install` step only takes about 30sec

Maybe I use the wrong way to init. Please give me another chance to try again

**Comment by LaoshuBaby** [comment]
> 
> 
> > > If use npm install, the install step may take more than 5 minutes, that will cause github kill this run.
> > 
> > 
> > I don't think so - our CI builds are already doing this, and the `npm install` step only takes about 30sec
> 
> Maybe I use the wrong way to init. Please give me another chance to try again

https://github.com/LaoshuBaby/name-suggestion-index/runs/2821896581

https://github.com/LaoshuBaby/name-suggestion-index/blob/30cc7678928d69c223f321aa48db61281f882482/.github/workflows/release.yml#L20-29

![图片](https://user-images.githubusercontent.com/42690037/121927833-18bdf480-cd72-11eb-9d02-7bdd7a4e65a7.png)
